src/tools/asm2wasm.cpp

```diff
@@ -101,9 +103,15 @@ int main(int argc, const char *argv[]) {
            [&legalizeJavaScriptFFI](Options *o, const std::string &) {
              legalizeJavaScriptFFI = false;
            })
-      .add("--debuginfo", "-g", "Emit names section and debug info (for debug info you must emit text, -S, for this to work)",
+      .add("--debuginfo", "-g", "Emit names section in wasm binary (or full debuginfo in wast)",
            Options::Arguments::Zero,
            [&](Options *o, const std::string &arguments) { passOptions.debugInfo = true; })
+      .add("--binarymap-file", "-bm", "Emit binary map (if using binary output) to the specified file",
+           Options::Arguments::One,
+           [&binaryMapFile](Options *o, const std::string &argument) { binaryMapFile = argument; })
+      .add("--binarymap-url", "-bu", "Use specified string as binary map URL",
```

Comment:
This parameter defines the sourceMappingURL property that will be written in the .wasm custom section.